test/functional/api/v2/hosts_controller_test.rb

```diff
@@ -89,6 +89,38 @@ def expect_attribute_modifier(modifier_class, args)
     assert_includes(ip_addresses, "127.13.0.1")
   end
 
+  test "should get parameters from index" do
+    last_record.parameters = [HostParameter.new(name: 'foo', value: 'bar')]
+    last_record.save!
+    get :index, include: ['parameters']
+    assert_response :success
+    assert_not_nil assigns(:hosts)
+    hosts = ActiveSupport::JSON.decode(@response.body)
+    parameters = hosts['results'].map { |host| host['parameters'] }.flatten
+    parameter = parameters.select { |param| param['name'] == 'foo' }.first
+    refute parameters.empty?
+    assert_equal parameter['value'], 'bar'
+  end
+
+  test "should get all_parameters from index" do
+    hostgroup = FactoryGirl.create(:hostgroup, :with_parent, :with_domain, :with_os)
+    hostgroup.group_parameters = [GroupParameter.new(name: 'foobar', value: 'baz')]
+    hostgroup.save!
+    last_record.parameters = [HostParameter.new(name: 'foo', value: 'bar')]
+    last_record.save!
```

Comment:
This isn't doing anything, it can be removed (it probably should have been after the hostgroup set, but it won't work as last_record retrieves a new copy every time it's called).